views/components/BarGraphTooltip/index.jsx

```diff
@@ -2,29 +2,34 @@ import React from 'react'
 import { TOTAL_LABEL } from '../../../constants'
 import { isTargetShown, siteTooltipContent } from './helpers'
 
+import './BarGraphTooltip.css'
+
 const BarGraphTooltip = ({ active, payload, label, studyTotals }) => {
   if (!active || !payload?.length) {
     return null
   }
 
   return (
-    <div>
-      <div>
-        <div>{label}</div>
-        <div>{isTargetShown(payload) ? 'Value / Target' : 'Value'}</div>
+    <div className="BarGraphTooltip">
+      <div className="BarGraphTooltip_Header">
+        <div className="BarGraphTooltip_Label">{label}</div>
+        <div className="BarGraphTooltip_Value">
+          {isTargetShown(payload) ? 'Value / Target' : 'Value'}
+        </div>
       </div>
       {siteTooltipContent(payload, studyTotals[label]).map(
         ({ labelColumn, valueColumn }) => {
           return (
             <div
+              className="BarGraphTooltip_Row"
               key={`${labelColumn}-${valueColumn}`}
               style={{
```

Comment:
Do we have `classname` still? We could use that instead of `style` here, which would allow us to keep the border styles in our CSS

Reply:
[094aa13](https://github.com/AMP-SCZ/dpdash/pull/529/commits/094aa13a95831d13d66ca3fc781ced6dbe96a4c7)